src/scoring.js

```diff
@@ -64,17 +64,17 @@ function bayesFactors(nlogpvals, cap=true) {
 }
 
 /**
- * Convert an array of raw per-element probability scores into posterior probabilities, dividing by the sum of all
- *   elements in the range.
+ * Convert an array of Bayes factors into posterior probabilities by normalizing each factor
```

Comment:
Given the simplicity of this method, is there a strong reason to couple it specifically and only to Bayes factors? It seems that other scoring methods might also have a need to be normalized- leaving room for a codebase to grow is a delicate balance.

(I realize that naming the method `posteriorProbabilities` probably undercuts my case, but, well, that's irony for you...)

Anyway, let me know when the PR and changes are ready. I have a few more edits to docs and examples and will hold off pushing them upstream. See personal repo:
https://github.com/abought/gwas-credible-sets/commit/17465c17463c8a017a80900bca93ddf8c5599d35

Reply:
Actually you could just push your changes and I'll rebase on top of them, I don't think I will likely change much more, everything looks great. :) 

It's a simple method but unfortunately it only really makes sense to convert Bayes factors to posterior probabilities (and it's actually specific to our method due to the assumptions made for that to be the case, as I understand it.) 

Maybe a more generic function could be substituted entirely. This function is basically [normalize](http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.normalize.html) using L1-norm (we sum the values, L1-norm sums the absolute values, but all of our values are always positive anyway.)

Reply:
> Actually you could just push your changes and I'll rebase on top of them, I don't think I will likely change much more, everything looks great. :)

Done. 

Only other pending change: I'm separating the `findCredibleSet` method out from the marking options (boolean, scaled, etc).  Eg: `markBoolean(credibleSet)`

It turns out there is a good use case for marking / visualizing the data in multiple ways on a plot, and it seemed silly to keep doing the underlying credible set calc every time.